js/ai/src/formats/types.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 import type { JSONSchema } from '@genkit-ai/core';
-import type { GenerateResponseChunk } from '../generate.js';
+import type { GenerateResponseChunk } from '../generate/chunk.js';
```

Comment:
It was complaining about re-exports so I got it from the original definition instead of the re-export